Owncloud iOs Client/DataBase/Queries/ManageDB.m

```diff
@@ -955,4 +955,30 @@ + (void) updateDBVersion15To16{
     
 }
 
+///-----------------------------------
+/// @name Update Database version with 16 version to 17
+///-----------------------------------
+
+/**
+ * Changes:
+ *
+ * Alter users table, added new field to store the sorting choice for showing folders/files in file list.
+ *
+ */
++ (void) updateDBVersion16To17{
+    
+    FMDatabaseQueue *queue = Managers.sharedDatabase;
+    
+    [queue inTransaction:^(FMDatabase *db, BOOL *rollback) {
+        BOOL correctQuery=NO;
+        
+        correctQuery = [db executeUpdate:@"ALTER TABLE users ADD sorting_type BOOL DEFAULT 0"];
```

Comment:
Do not use BOOL use INTEGER in order to use an enum to show more than two order possibilities
 Add a line note